tests/dao_tests/test_dv_order_dao.py

```diff
@@ -83,6 +84,39 @@ def test_enumerate_tracking_status(self):
         )
         self.assertEqual(status, OrderShipmentTrackingStatus.IN_TRANSIT)
 
+    def test_from_client_json(self):
+        # pylint: disable=unused-variable
+        req_payload = self.send_post(
+            "SupplyRequest",
+            request_data=self.post_request,
+            expected_status=http.client.CREATED,
+        )
+        deliv_payload = self.send_post(
+            "SupplyDelivery",
+            request_data=self.post_delivery,
+            expected_status=http.client.CREATED,
+        )
+
+        existing_obj = self.dao.from_client_json(self.post_delivery, participant_id=self.participant.participantId)
```

Comment:
The typical pattern is to use something like `existing_obj` when we fetch a record from the DB and are comparing it to a new obj. This name seems a little misleading.